src/main/kotlin/no/nav/syfo/cronjob/behandlendeenhet/PersonBehandlendeEnhetQuery.kt

```diff
@@ -63,7 +63,8 @@ const val queryGetPersonerWithOppgaveAndOldEnhet =
         OR dialogmotekandidat = 't' 
         OR dialogmotesvar_ubehandlet = 't'
         OR ((aktivitetskrav = 'NY' OR aktivitetskrav = 'AVVENT') AND aktivitetskrav_stoppunkt > '2023-03-10')
-        OR behandlerdialog_ubehandlet = 't'
+        OR behandlerdialog_svar_ubehandlet = 't'
+        OR behandlerdialog_ubesvart_ubehandlet = 't'
```

Comment:
Går fint at å bare skrive "OR behandlerdialog_ubesvart_ubehandlet", men kanskje greit å følge eksisterende pattern (men synes kanskje vi bør bli enige om *en* måte å skrive slikt i queries på, for det er mange varianter som er gyldig postgres-sql. Jeg foretrekker varianten som er nærmest Kotlin/Java).